modules/doc/content/getting_started/installation/install_conda_moose.md

```diff
@@ -1,66 +1,36 @@
 ## Install MOOSE Conda Packages id=moosepackages
 
-!alert! note
-If conda was installed previously, and the [install_miniconda.md] instructions were skipped,
-we highly encourage the use of [Mamba](https://github.com/mamba-org/mamba) to install
-packages, due to speed. We recommend that you perform:
-
-```
-conda deactivate
-conda install mamba
-```
-
-in your base environment.
-!alert-end!
-
-Begin by creating the moose environment within conda, and attempt to activate it:
+Before we create our virtual conda environment, we first need to initialize mamba.
+For this, execute the following command and +restart your session+.
 
 ```bash
-conda create --name moose -q -y
-conda activate moose
-```
-
-During activation, you may be presented with an error such as the following:
-
-```pre
-CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
-To initialize your shell, run
-
-    $ conda init <SHELL_NAME>
-
-Currently supported shells are:
-  - bash
-  - fish
-  - tcsh
-  - xonsh
-  - zsh
-  - powershell
-
-See 'conda init --help' for more information and options.
-
-IMPORTANT: You may need to close and restart your shell after running 'conda init'.
+mamba init
 ```
 
-This error is trying to exlain how to initialize Conda for the first time for your given shell environment. Different operating systems use different shells. To understand what shell you are using, perform the following:
+Next, we create a moose environment, and attempt to activate it:
```

Comment:
Next, create a unique conda environment for moose, named `moose`, and attempt to activate it: